### ISSUE TITLE
Tabbing and selecting

### ISSUE THREAD

**Issue by xythh** [xythh/ann2html#2]
A featured I was trying to implement but could never figure out was how to autoselect a element that you tab to. This would make mining even easier because you could tab through the file, it would select the thing you tabbed to(which would be the word you highlighted on your kindle) and then you can set up a yomichan keybinding to scan selected text and you could mine without ever moving your mouse.

If anyone has any idea how to implement this, I would greatly appreciate the help.

TLDR: Tab through the file currently, You can see how your browser outlines the word you tabbed to, how can I make it so your browser selects that word when you tab to it.

**Comment by xythh** [comment]
Doesn't seem to be a clean way to implement this, scrolling isn't too bad.
